store/sqlstore/group_supplier.go

```diff
@@ -919,25 +934,25 @@ func (s *SqlSupplier) ChannelMembersToRemove(ctx context.Context, hints ...store
 	return result
 }
 
-func (s *SqlSupplier) groupsByTeamBaseQuery(t selectType, teamID string, opts model.GroupSearchOpts) squirrel.SelectBuilder {
+func (s *SqlSupplier) groupsBySyncableBaseQuery(st model.GroupSyncableType, t selectType, syncableID string, opts model.GroupSearchOpts) squirrel.SelectBuilder {
 	selectStrs := map[selectType]string{
 		selectGroups:      "ug.*",
 		selectCountGroups: "COUNT(*)",
 	}
 
 	query := s.getQueryBuilder().
 		Select(selectStrs[t]).
-		From("GroupTeams gt").
+		From(fmt.Sprintf("Group%ss gt", st.String())).
```

Comment:
I don't like it I would prefer something more explicit like:
```go
var tableToQuery string
if st == model.GroupSyncableTypeTeam {
    tableToQuery = "GroupTeams gt" // probably here is better to use "g" instead of "gt" because you are selecting groupTeams or Group Channels
}  else {
    tableToQuery = "GroupChannels gt"
}
```